src/timesync/timesyncd.c

```diff
@@ -132,6 +132,8 @@ int main(int argc, char *argv[]) {
         if (r < 0)
                 log_warning_errno(r, "Failed to parse configuration file: %m");
 
+        assert_se(manager_parse_fallback_string(m, NTP_SERVERS) >= 0);
```

Comment:
humm, what? Except in test cases we don't use assert_se() for catching OOM errors, and this might result in them. This needs to be done properly.